packages/ckeditor5-utils/tests/keyboard.js

```diff
@@ -232,6 +280,52 @@ describe( 'Keyboard', () => {
 			} );
 		} );
 
+		describe( 'on iOS', () => {
```

Comment:
The tests you added won't protect us from regressions. You can test it on mac.

Open terminal in your main ckeditor5 directory. Execute following 2 commands to undo your fix for logic (but to keep your tests).

```
git fetch --all
git checkout origin/master packages/ckeditor5-utils/src/keyboard.ts
```

Run tests, you can run just this single file to make it simpler:

```
yarn test -f utils/keyboard -w
```

## Expected

The tests should fail ❌ because there's no fix from this PR applied.

## Actual

The tests are passing.

## Reason

When running tests on mac, the `env.isMac` is set to true so it works due to the old logic:

[https://github.com/ckeditor/ckeditor5/blob/a8b1e91ba6855507b9d703a7e55076a05b2159be/packages/ckeditor5-utils/src/keyboard.ts#L124](https://github.com/ckeditor/ckeditor5/blob/a8b1e91ba6855507b9d703a7e55076a05b2159be/packages/ckeditor5-utils/src/keyboard.ts#L124)

That checks isMac.

In order to fix it you need to stub both `isMac` and `isiOS` (making sure that mac is false for ios tests, **and that ios is false for mac tests**).